### ISSUE TITLE
Support for Visual Basic 6.0 (VB6) or Visual Basic for Applications (VBA)

### ISSUE THREAD

**Issue by King555a** [alecthomas/chroma#762]
**What problem does this feature solve?**

I was informed that the software Gitea, which I use, uses Chroma and when I have a Visual Basic 6.0 project in my repository (and I have several), there is no syntax highlighting for VB6 files.

**What feature do you propose?**

Syntax highlighting for Visual Basic 6.0 (VB6) or Visual Basic for Applications (VBA). They are two different languages, but have the same syntax and so need the same highlighting.

I can post more information if this is something which can be done for future versions.

**EDIT:** Also VBS (VBScript/Visual Basic Script) has the same syntax. So you can cover three languages with one highlighter. 😉

**Comment by King555a** [comment]
I wonder why QBasic from 1991 is supported, but VBA, which is still part of modern MS Office, not.

**Comment by alecthomas** [comment]
Because nobody has put the effort into creating a lexer. It's XML, it's straightforward.

**Comment by King555a** [comment]
So if someone provides the necessary file, you would officially integrate it into Chroma?

**Comment by alecthomas** [comment]
Of course! You can use the CLI binaries on the release page to test it, so don't even need to know Go.
